backend/src/main/java/meetup/connect/common/exception/ControllerExceptionHandler.java

```diff
@@ -14,4 +20,31 @@ public ResponseEntity<ErrorResponse> handleException(MeetUpException ex) {
     ErrorResponse error = new ErrorResponse(ex.getMeetUpError(), LocalDateTime.now());
     return new ResponseEntity<>(error, ex.getMeetUpError().getHttpStatus());
   }
+
+  @ExceptionHandler(MethodArgumentNotValidException.class)
+  public ResponseEntity<ErrorResponse> handleMissingParams(MethodArgumentNotValidException ex) {
+    String errorMessage = ex.getBindingResult().getAllErrors().get(0).getDefaultMessage();
+    ErrorResponse error =
+        new ErrorResponse(
+            errorMessage, HttpStatus.BAD_REQUEST, ex.getStatusCode().value(), LocalDateTime.now());
```

Comment:
issue: a potential HTTP status code mismatch

Reply:
nit: let's create a constructor for an error message and an HTTP status code, something like the example below: 
`ErrorResponse(String message, HttpStatus httpStatus)` 
- you can get a code from the `HttpStatus` enum e.g. https://github.com/spring-projects/spring-framework/blob/10e8039de86e48db931f5b0d3773ee8ecfa62f88/spring-web/src/main/java/org/springframework/http/HttpStatus.java#L179, there is the `HttpStatus::value` method, it will extract the integer value of the requested HTTP code
- `time` is `LocalDateTime.now()` for most of the time, looks redundant